egs/wsj/s5/steps/cleanup/make_utterance_graph.sh

```diff
@@ -105,12 +105,13 @@ while read line; do
       utils/make_unigram_grammar.pl | fstcompile |\
       fstarcsort --sort_type=ilabel > $wdir/G.fst || exit 1;
   else
-    echo $words | awk -v voc=$lang/words.txt -v oov="$oov_txt" '
-      BEGIN{ while((getline<voc)>0) { invoc[$1]=1; } } {
-      for (x=1;x<=NF;x++) {
-      if (invoc[$x]) { printf("%s ", $x); } else { printf("%s ", oov); } }
-      printf("\n"); }' > $wdir/text
-    ngram-count -text $wdir/text -order $ngram_order "$srilm_options" -lm - |\
+     echo $words > $wdir/text
+     split -b 200000 $wdir/text $wdir/xxx
```

Comment:
... also, I see that you have moved the dealing-with-OOVs into the ngram-count process.
This would be OK, except it's extremely inefficient to deal with the whole word-list on each utterance each time, if the utterances are short.  The original script had this problem too, as the wordlist was read by awk- I did not check it sufficiently when it was originally committed.  
The while loop is done <$text.  A much-more-efficient way to do this would be to pipe into the while loop, a command involving sym2int.pl (with the appropriate options to map unknown words) and then into int2sym.pl.